### PR TITLE
Add smoke test for Microshift loadbalancer controller

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

**Comment by pliurh** [comment]
@pacevedom PTAL

File: ci-operator/step-registry/openshift/microshift/e2e/loadbalancer-smoke-test/openshift-microshift-e2e-loadbalancer-smoke-test-commands.sh [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+IP=$(LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute instances describe "${INSTANCE_PREFIX}" --format='value(networkInterfaces.accessConfigs[0].natIP)')
+
+set +ex
+retries=3
+backoff=3s
+
+for try in $(seq 1 "${retries}"); do
+  echo "Attempt: ${try}"
+  echo "Running: curl -vk ${IP}:5678"
+  RESPONSE=$(curl -vk ${IP}:5678 2>&1)
```
Comment (pacevedom) [inline ci-operator/step-registry/openshift/microshift/e2e/loadbalancer-smoke-test/openshift-microshift-e2e-loadbalancer-smoke-test-commands.sh:100]: Does it help if you use `curl -I` instead? That will only print the headers and the status code is on the first line.

File: ci-operator/step-registry/openshift/microshift/e2e/loadbalancer-smoke-test/openshift-microshift-e2e-loadbalancer-smoke-test-commands.sh [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  RESULT=$?
+  echo "Exit code: ${RESULT}"
+  echo -e "Response: \n${RESPONSE}\n\n"
+  if [ $RESULT -eq 0 ] && echo "${RESPONSE}" | grep -q -E "HTTP.*200 OK"; then
```
Comment (pacevedom) [inline ci-operator/step-registry/openshift/microshift/e2e/loadbalancer-smoke-test/openshift-microshift-e2e-loadbalancer-smoke-test-commands.sh:104]: Is it more robust if you skip the ` OK` part?

Comment (pliurh) [reply ci-operator/step-registry/openshift/microshift/e2e/loadbalancer-smoke-test/openshift-microshift-e2e-loadbalancer-smoke-test-commands.sh]: fixed

**Review by pacevedom — APPROVED** [review]
/lgtm

**Review by pacevedom — APPROVED** [review]
/lgtm

**Pull request merged**